.buildkite/publish/buildkite-entry.sh

```diff
@@ -27,6 +27,17 @@ echo $BUILDKITE_TAG
 echo $BUILDKITE_SOURCE
 echo $UPDATE_STUDIO # TODO can we to remove this?
 
+# Make sure docker instances are stopped to avoid the following flaky errors
+# `Bind for 0.0.0.0:27017 failed: port is already allocated`
+DOCKER_IDS=$(docker ps -q)
+echo $DOCKER_IDS
+if [ -v ${DOCKER_IDS} ]; then
+  echo "Did not find a docker instance running. We're good!"
+else
+  echo "Found docker instance(s) running. Let's stop them!"
+  docker kill $DOCKER_IDS
```

Comment:
Let's try `docker-compose down -v` then

Note, the docker-compose Buildkite plugin is supposed to already do that, but when this happens it might be a compute instance reused from another pipeline that doesn't do that (my best theory so far)

Reply:
But using `docker-compose down -v` means we need a docker-compose file, and I think here it's safer to stop any running instance we don't even know if they were started from this pipeline

Reply:
Just added that! Thanks 👍🏼